src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderFlowValidator.java

```diff
@@ -146,14 +151,14 @@ private Future<Void> validateLocationRestrictions(CompositePoLine poLine, List<F
     Set<Fund> restrictedFunds = funds.stream().filter(Fund::getRestrictByLocations).collect(Collectors.toSet());
 
     if (restrictedFunds.isEmpty()) {
-      logger.info("validateLocationRestrictions:: Funds is not restricted by locations");
+      logger.info("validateLocationRestrictions:: No funds are restricted by locations");
       return Future.succeededFuture();
     }
 
     return extractRestrictedLocationIds(poLine, restrictedFunds, requestContext)
       .compose(restrictedLocations -> {
         if (restrictedLocations.isEmpty()) {
-          logger.info("validateLocationRestrictions:: restricted locations is not found for poLineId '{}'", poLine.getId());
+          logger.info("validateLocationRestrictions:: no restricted locations found for poLineId '{}'", poLine.getId());
```

Comment:
Lets start with Capital letters, I see all places you started with Capital letters